### PR TITLE
Emoji reactions on posts

### PR DESCRIPTION
**Pull request by oetherington**
[Relevant Figma](https://www.figma.com/file/d09iVzNSpZp9GyWGcfkWmX/Q2-2023?node-id=1791%3A92368&mode=dev)

This PR adds post-level emoji reactions. Most of the line count just comes from moving around some frontend code.

Also included in this PR is the experimental change to make upvotes green.

Screenshot 2023-09-07 at 01 00 36

Screenshot 2023-09-07 at 01 00 57

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205428221373383) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/votes/EAEmojisVoteOnComment.tsx [hunk 1]
```diff
@@ -27,299 +17,8 @@ const styles = (theme: ThemeType): JssStyles => ({
       },
     },
   },
-  button: {
```
Comment (oetherington) [inline packages/lesswrong/components/votes/EAEmojisVoteOnComment.tsx:30]: All the code deleted in this file just moved into `EAReactsSection` for reuse elsewhere

File: packages/lesswrong/components/votes/PostsVote.tsx [hunk 2]
```diff
@@ -1,149 +1,39 @@
-import { Components, registerComponent } from '../../lib/vulcan-lib';
```
Comment (oetherington) [inline packages/lesswrong/components/votes/PostsVote.tsx:1]: All the code deleted in this file just moved into `PostsVoteDefault`

**Comment by oetherington** [comment]
FYI @agnesstenlund

**Comment by agnesstenlund** [comment]
some of the emojis also look even smaller on posts than comments 😮 
Screenshot 2023-09-07 at 12 03 33
Screenshot 2023-09-07 at 12 03 46

the delta one (and agree/disagree) looks like the right size, but the rest look really small 
Screenshot 2023-09-07 at 12 04 09

**Comment by agnesstenlund** [comment]
looking more closely at this dropdown I also think it'd be neat to show the icons a bit bigger here too (for both comments and post). Especially the last two look really tiny
Screenshot 2023-09-07 at 12 24 50

**Comment by oetherington** [comment]
@agnesstenlund I just pushed some commits that should fix the remaining things you mentioned. Here's the dropdown with the increased emoji size:
Screenshot 2023-09-11 at 12 00 54

**Comment by agnesstenlund** [comment]
looks great, thanks! I pushed a commit with some tiny tweaks, should be good to go now from a design perspective

File: packages/lesswrong/themes/defaultPalette.ts [hunk 3]
```diff
@@ -523,5 +523,9 @@ export const defaultComponentPalette = (shades: ThemeShadePalette): ThemeCompone
   },
   warning: {
     main: "#ff9800",
-  }
+  },
+  greenUpvote: {
+    main: "#61CD5A",
+    light: "#70d26a",
+  },
```
Comment (s-cheng) [inline packages/lesswrong/themes/defaultPalette.ts:530]: I feel like this green is too light - it's pretty hard to see, esp in light mode (also I did a quick colorblindness test and this green looks very similar to the grey for the most common form of colorblindness) @agnesstenlund FYI

Screen Shot 2023-09-11 at 4 15 05 PM

File: packages/lesswrong/lib/collections/posts/fragments.ts [hunk 4]
```diff
@@ -48,6 +48,7 @@ registerFragment(`
     voteCount
     baseScore
     extendedScore
+    emojiReactors
```
Comment (s-cheng) [inline packages/lesswrong/lib/collections/posts/fragments.ts:51]: nbd but since this is in `PostsBase` you prob don't need to add it elsewhere?

**Review by s-cheng — APPROVED** [review]
Excited to have this!

A small piece of UI feedback: I feel like we could use more space between the reacts (or less space between the emoji and the number) @agnesstenlund FYI

Screen Shot 2023-09-11 at 4 35 53 PM

**Comment by jpaddison3** [comment]
Sorry, I missed this earlier. Is there any chance we could experiment with the green upvote color only for beta users?

**Comment by agnesstenlund** [comment]
> A small piece of UI feedback: I feel like we could use more space between the reacts (or less space between the emoji and the number) @agnesstenlund FYI

agree, thanks for calling it out! Let's increase the space between the emojis (that way it'll have more breathing room when they're selected too)

also, realised the upvote in the top left by the post header is now much tighter–could we make that look more like the old one?

**Comment by oetherington** [comment]
All the above comments should now be fixed, except for the contrast on green votes. We could choose something now, or, since they're now beta gated, maybe we just merge and fix it later?

**Comment by agnesstenlund** [comment]
I'd vote in favour of merging since the green tick is only on beta 👍  

also checked the "main" and "light" and they are almost identical colors–so the faded color must be an opacity thing only:
Screenshot 2023-09-14 at 13 09 48

**Comment by agnesstenlund** [comment]
actually nvm, changed it now :)

**Pull request merged**